### DISCUSSION TITLE
Error on build

### DISCUSSION THREAD

**Discussion by danillo-hilbs** [nuxt/nuxt#17069] [Questions]
I cant run my build anymore, i try cloning repository again, clean node_modules. the same build runs in others developers  pc, but i ever got this error

![image](https://user-images.githubusercontent.com/34604307/184373987-36ca689b-8d92-4b0f-8d4b-4d5db792e58a.png)

**Comment by pxtrez** [comment]
Remove `lang="ts"` from `pages/supernova.vue`, it worked for me
